pkg/core/client/git.go

```diff
@@ -0,0 +1,62 @@
+/**
+ * This package is intended for git access from FuseML client
+ * It needs GITEA_URL set as environment variable.
```

Comment:
Well, in the end, these values have to be the same, be it client or server. So I find it actually correct to share a config package

Reply:
I might change the env var names to just GIT_* if you think it is better, but that's probably not your main point

Reply:
> Well, in the end, this value have to be the same, be it client or server. So I find it actually correct to share a config package

Conceptually, you wouldn't share a config package to be used by both client and server, because they're different entities and running as different processes. It just seems convenient in this case, because we're using hard-coded values for username/password in both client and server and they need to be coded somewhere common. This is more about importing in the client only what is needed by the client, and some of that stuff in the config package is only needed by the server.

Anyway, I agree this is more of a nitpick then something that needs changing, so you can leave it as is.

Reply:
So if we get to the point when credentials for "fuseml user" are returned from the server so that client needs to use them explicitly, we would change this dependency indeed. Something for later then.

Reply:
This would be https://github.com/fuseml/fuseml/issues/62